Automattic/jetpack
# Fix/soft disconnection deprecation

Fixes issue in connection consumers that were checking `$plugin->is_enabled()`. (added in #23991)

The deprecation of soft disconnection made this method superfluous, but for BC reasons it was kept. But it was broken.

#### Changes proposed in this Pull Request:

* Fix plugin->is_enabled method

#### Jetpack product discussion


p1651023531345719-slack-C02AVAR9B

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Connect Jetpack
* Activate Boost and check it working as expected
* Activate woo-commerce payments and make sure it finds itself connected
* Test connecting, using and disconnecting different combinations of plugins (Jetpack and Boost, Jetpack and WCPayments, Bakcup and WCPayments, Backup and Boost, etc...)
* After deactivating a plugin (while other connected plugin is still active), check we have the right information on WPCOM
 * `jetpack_connected_active_plugins`
 * `jetpack_package_versions`
